ui/lib/pki/addon/components/page/pki-tidy-form.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import errorMessage from 'vault/utils/error-message';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import { waitFor } from '@ember/test-waiters';
+import { tracked } from '@glimmer/tracking';
+
+import PkiTidyModel from 'vault/models/pki/tidy';
+import RouterService from '@ember/routing/router-service';
+
+interface Args {
+  tidy: PkiTidyModel;
+}
+
+export default class PkiTidyForm extends Component<Args> {
+  @service declare readonly router: RouterService;
+
+  @tracked errorBanner = '';
+  @tracked invalidFormAlert = '';
+
+  returnToConfiguration() {
+    this.router.transitionTo('vault.cluster.secrets.backend.pki.configuration.index');
+  }
+
+  @action
+  updateSafetyBuffer({ goSafeTimeString }: { goSafeTimeString: string }) {
+    this.args.tidy.safetyBuffer = goSafeTimeString;
+  }
+
+  @task
+  @waitFor
+  *save(event: Event) {
+    event.preventDefault();
+    try {
+      yield this.args.tidy.save();
```

Comment:
what do you think about passing in the information about which post url to use directly to the save method? we did something similar in `pki-import-pem-bundle.ts` [here](https://github.com/hashicorp/vault/blob/e97371680c0d9a5da44d720a55376c9a02008fcc/ui/lib/pki/addon/components/pki-import-pem-bundle.ts#L64) that way the model doesn't have to keep track of it as an attribute, and only the adapter/save method care about it. 

making it a component arg also makes it extensible so we can reuse this form component for the auto-tidy config work